### PR TITLE
Revert PR #12

### PR DESCRIPTION
**Pull request by hakonhagland**
Refer to https://github.com/perl-ide/App-perlvars/issues/17#issue-2528477424

Revert the temporary fix in PR #12 as the temporary fix may not be needed after Test::Vars has been updated.

### CODE REVIEW

**Review by oalders — CHANGES_REQUESTED** [review]
This looks great. Can we also bump the minimum version of Test::Vars to the latest?

**Comment by hakonhagland** [comment]
> Can we also bump the minimum version of Test::Vars to the latest?

@oalders See last commit. Does it look right?

File: cpanfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ requires "Moo" => "0";
 requires "PPI::Document" => "0";
 requires "Path::Tiny" => "0";
 requires "Pod::Usage" => "0";
-requires "Test::Vars" => "0";
```
Comment (oalders) [inline cpanfile:9]: I think this line is enough. We don't need to set the version in `dist.ini` and the other changes will be generated by `Dist::Zilla` as well. I could have been a lot clearer about that. My bad!

Comment (hakonhagland) [reply cpanfile:9]: No problem! I have reverted the other changes :)

**Comment by oalders** [comment]
Thank you so much @hakonhagland! I've squashed and merged at the command line via 78461f1d5500ed

**Pull request closed without merging**